phylanx/ir/node_data.hpp

```diff
@@ -114,6 +114,18 @@ namespace phylanx { namespace ir
           : data_(storage1d_type(values.size(), values.data()))
         {
         }
+        explicit node_data(std::vector<std::vector<T>> const& values)
+        {
+          const std::size_t nx = values.size();
+          const std::size_t ny = values[0].size();
+          storage2d_type store{nx,ny};
+          for(std::size_t i=0;i<nx;i++) {
+            for(std::size_t j=0;j<ny;j++) {
+              store(i,j) = values[i][j];
+            }
+          }
+          operator=(std::move(store));
+        }
```

Comment:
Could you please stick to the formatting used in this file (i.e. 4 characters of whitespace used for indentation)?